### PR TITLE
README: Update helm repo url to Scala Steward gh-pages url

### PR DESCRIPTION
**Pull request by MPV**
Fixing README so it reflects the new URL (after the move of this repo into scala-steward-org in #3).

### CODE REVIEW

**Comment by MPV** [comment]
@jsoref @MaT1g3R 😊

File: README.md [hunk 1]
```diff
@@ -1,27 +1,11 @@
...
 ```sh
-helm repo add garnercorp https://garnercorp.github.io/charts
-helm search repo garnercorp
-helm install my-release garnercorp/<chart>
+helm repo add scala-steward-org https://scala-steward-org.github.io/charts
+helm search repo scala-steward-org
+helm install my-release scala-steward-org/<chart>
 ```
-
-# License
```
Comment (jsoref) [inline README.md:13]: In principle removing license blobs is scary. Please explain.

I'm fine with everything else, but this I'll have to check.

Comment (MPV) [reply README.md:13]: Ah, no worries, feel free to keep of course. I just naively assumed it was an unintended remnant from before the move. Happy to add it back, I have no particular view on the topic.

Comment (MPV) [reply README.md:13]: I added it back now — looking better 👍? 😊

Comment (jsoref) [reply README.md:13]: The thing about the license, is it's in more than one place: https://github.com/scala-steward-org/charts/blob/master/LICENSE

On the bright side, it's consistent 😄 .

**Pull request merged**